mod/05_standard/set/type/uri.rb

```diff
@@ -0,0 +1,11 @@
+format do
+  include Phrase::Format
+
+  view :core do |args|
+    build_link _render_raw(args), render_title(args) || card.name
+  end
+
+  view :uri do |args|
+    build_link _render_raw(args)
```

Comment:
I think we just have the wrong name for the view.  The intention is to create a link whose link text is the link rather than the cardname/title.  Maybe this should be done with an inclusion options instead, then other types would just ignore it.